test/wdio/lifecycle-update/cmp-a.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('lifecycle-update', function () {
+  beforeEach(() => {
+    render({
+      template: () => (
+        <>
+          <ol id="output"></ol>
+          <hr />
+          <lifecycle-update-a></lifecycle-update-a>
+        </>
+      ),
+    });
+  });
+
+  it('fire load methods in order', async () => {
+    await $('#output').waitForStable();
```

Comment:
@christian-bromann Would it make more sense to use `waitForChanges` here from `@wdio/brwoser-runner/stencil`, or does it not matter?

Reply:
`waitForChanges` doesn't work and I need to remove it from the package. I initially thought I could port the functionality but it doesn't look like it.

Reply:
wondering about this too 🤔